resolwe_bio/processes/workflows/bbduk_star.py

```diff
@@ -199,11 +199,11 @@ class ChimericReadsOptions:
                 """Chimeric reads options."""
 
                 chimeric = BooleanField(
-                    label="Detect chimeric and circular alignments [--chimOutType SeparateSAMold]",
+                    label="Detect chimeric and circular alignments [--chimOutType]",
                     default=False,
                     description="To switch on detection of chimeric (fusion) alignments (in addition "
```

Comment:
Same comment about the description of `chimeric` belnging to `chim_segment_min`.